zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/approval/ApproveTask.java

```diff
@@ -329,18 +323,77 @@ private static ContentDetailResolver getDefaultContentDetailResolver() {
 
     private void validate() {
         if (collection == null) {
-            throw new IllegalArgumentException("approval task unsuccesful: collection required but was null");
+            throw new IllegalArgumentException("approval task unsuccessful: collection required but was null");
         }
         if (collection.getDescription() == null) {
-            throw new IllegalArgumentException("approval task unsuccesful: collection.description required but was " +
+            throw new IllegalArgumentException("approval task unsuccessful: collection.description required but was " +
                     "null");
         }
         if (session == null) {
-            throw new IllegalArgumentException("approval task unsuccesful: as session required but was null");
+            throw new IllegalArgumentException("approval task unsuccessful: as session required but was null");
         }
         if (StringUtils.isEmpty(session.getEmail())) {
-            throw new IllegalArgumentException("approval task unsuccesful: as session.email required but was null/empty");
+            throw new IllegalArgumentException(
+                    "approval task unsuccessful: as session.email required but was null/empty");
         }
-        info().data("collectionId", collection.getDescription().getId()).log("approval task: validation sucessful");
+        info().data("collectionId", collection.getDescription().getId()).log("approval task: validation successful");
+    }
+
+    protected void uploadNewEndpoint(Collection collection, CollectionReader collectionReader) throws ZebedeeException, IOException {
+        if (Configuration.isUploadNewEndpointEnabled()) {
+            uploadWhitelistedFiles(collection, collectionReader);
+        }
+    }
+    
+    protected void uploadWhitelistedFiles(Collection collection, CollectionReader collectionReader) throws ZebedeeException, IOException {
+        for (String uri : collectionReader.getReviewed().listUris()) {
+            if (uri.endsWith(".csv") || uri.endsWith(".xlsx")) {
```

Comment:
Where has the check for csdb and xls files gone as these need adding to the bucket also